src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -190,10 +199,27 @@ protected static function bootTraits()
     {
         $class = static::class;
 
+        static::$traitInitializers[$class] = [];
+
         foreach (class_uses_recursive($class) as $trait) {
             if (method_exists($class, $method = 'boot'.class_basename($trait))) {
                 forward_static_call([$class, $method]);
             }
+            if (method_exists($class, $method = 'initialize'.class_basename($trait))) {
+                static::$traitInitializers[$class][] = $method;
+            }
+        }
+    }
+
+    /**
+     * Initialize any initializable traits on the model.
```

Comment:
Perhaps stick to the boot doc:

> Initialize all of the initializable traits on the model.

Reply:
I don't follow completely what you are suggesting. :)

Reply:
https://github.com/ralphschindler/laravel-framework/blob/0cab06b58765a70dcb7af1e4758e538190b375ee/src/Illuminate/Database/Eloquent/Model.php#L193-L198

It's whatever; just OCD I don't know.